shared/search/services-filter/index.js

```diff
@@ -79,7 +79,7 @@ const Service = ({service, selected, hovering, onHover, onSelect}) => {
         <Box
           style={{
             ...serviceTooltipStyle,
-            opacity: hovering ? 1 : 0,
+            display: hovering ? 'flex' : 'none',
```

Comment:
This gets rid of the transition on `opacity`, you might be able to set `pointerEvents: 'none'` here so it doesn't respond to hover

Reply:
Thanks for this, updated!